m4_sessions/1711-inning/Group_6.md

```diff
@@ -1,9 +1,23 @@
 Group Six
 
-Session Title: Description: Why should students come to this session?
+Session Title: Just because you can, doesn't mean you should: How to be an ethical technical professional.
+
+Description: Why should students come to this session?
+  As technology professional we will have to deal with privacy issues of users we are responsible for providing services for.  There are times where you will have to make a moral decision instead of a legal one.  How do you go about deciding if a legal technical action is moral or not?  Should companies take a moral stand on anything related to the security of their users information?
 
 Session Outcomes: What will students leave with or gain from this session?
+  An appreciation for ethical issues involving security and user data and a consideration for circumstances that, while legal, may not be moral.  They will be forced to consider several hypothetical situations where they may be tempted to behave unethically.
 
 Session Outline: Include the flow of the session and the time for each section.
+10 minutes - Icebreaker/Introductions
+10 minutes - Introduction (either a reading or a video)
+15 minutes - Break into groups (facilatated by student leads) and discuss a scenario
+30 minutes - Group discussion 
```

Comment:
Be specific about what the group is discussing AND how they will share...everyone?  Small groups?